.github/workflows/test.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
       TOXENV: "alldeps-withcov-posix,coverage-prepare,codecov-push,coveralls-push"
     strategy:
       matrix:
-          python-version: [3.5, 3.6, 3.7, 3.8, 3.9, pypy3]
+          python-version: [3.5, 3.6, 3.7, 3.8, 3.9, pypy-3.6, pypy-3.7]
```

Comment:
You should keep pypy3 here, in addition to adding pypy-3.6 and pypy-3.7.

Then after this release, we can remove pypy3 and keep pypy-3.6+

Reply:
It seems that `pypy3` refers to 3.6.  What do we need to keep it around for?

https://github.com/twisted/twisted/runs/1905976630?check_suite_focus=true
![image](https://user-images.githubusercontent.com/543719/108002608-23cc6800-6fbe-11eb-9f02-71fd9b5260ed.png)

Reply:
Yeah looks like specifying `pypy3` in GitHub actions installs pypy3 compatible with Python 3.6.12.
So I think your change is fine, not worth fighting with GitHub about this.

Reply:
This change is fine.